index.php

```diff
@@ -1,5 +1,8 @@
-<!DOCTYPE html>
+<?php
+    require('project2/fetch.php');
```

Comment:
rename folder project2 - it doesnt mean anything?

Reply:
done